static/sass/_patterns_carousel.scss

```diff
@@ -0,0 +1,74 @@
+@mixin juju-carousel {
+  .p-carousel {
+    position: relative;
+
+    .p-carousel__cell {
+      // height: 100%;
```

Comment:
I guess you can remove the commented out code

Reply:
:+1: